docker/ssl/authorization.js

```diff
@@ -29,8 +29,8 @@ function authenticate(s) {
 
         var pass = parsePackage(s, data);
 
-        if (!clientKey.length || pass !== clientKey) {
-            s.error('Cert CN (' + clientKey + ') does not match client password');
+        if (!clientKey.length || !clientKey.includes(pass) ) {
```

Comment:
Is there an option to use it like `endsWith` which is more precise for ignoring prefixes?